homeassistant/components/xiaomi_miio/.translations/en.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
We reworked our translations structure. Please rebase on  dev branch and rename the translations directory `.translations` to `translations`.

Also check the result by running the translation dev script again.

Reply:
Is it possible to rebase from the web interface of Github?

Reply:
Not the default interface I think.

Reply:
I have downloaded the github desktop client and I think I did a succesfull rebase